src/test/anovos/feature_recommender/test_feature_exploration.py

```diff
@@ -0,0 +1,177 @@
+import pandas as pd
+import pytest
+from anovos.feature_recommender.feature_exploration import (
+    list_all_industry,
+    list_all_usecase,
+    list_all_pair,
+    process_usecase,
+    process_industry,
+    list_usecase_by_industry,
+    list_industry_by_usecase,
+    list_feature_by_industry,
+    list_feature_by_usecase,
+    list_feature_by_pair,
+)
+from anovos.feature_recommender.featrec_init import model_download
+
+
+model_download()
+
+
+def test_list_all_industry():
+    test_df = list_all_industry()
+    assert len(test_df) > 0
+    assert "Industry" in test_df.columns
+    assert test_df.iloc[:, 0].nunique() == len(test_df)
+    for i in range(len(test_df)):
+        assert "," not in test_df.iloc[i, 0]
```

Comment:
Why are we checking if "," is not there in the Industry list 
What if a new industry being added has "," as part of its title?

Also Can I suggest something like checking for a couple of basic Industries if its available - like ones we feel important to have in mandatorily as a check.

Reply:
"," in here is to check if the input dataset used for Feature Exploration is the proper one. "," in the Industry/Usecase means that there are more than 1 Industry/Usecase in that particular cell.

> Also Can I suggest something like checking for a couple of basic Industries if its available - like ones we feel important to have in mandatorily as a check.
Sure, I will add it.